src/components/ui/testinomial-card2.tsx

```
@@ -0,0 +1,73 @@
+        <Link href='/'>
+            {
+                items?.length > 0 && items.map((el, idx) => {
+
+                    return (
+
+                        <div
+
+                       
+                        className={cn(" items-center justify-center md:p-4 w-44 md:w-60  relative overflow-hidden rounded-lg bg-foreground text-background px-2 md:px-8 py-3.5 transition-all duration-300 hover:scale-[1.02] hover:shadow-lg active:scale-95 cursor-pointer my-4 ", className ,
+                            el.big == true && "h-80"
+                        )}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)